lib/decompress/zstd_decompress_block.c

```diff
@@ -988,8 +992,11 @@ ZSTD_decodeSequence(seqState_t* seqState, const ZSTD_longOffset_e longOffsets)
         seq.offset = offset;
     }
 
-    seq.matchLength = mlBase;
+#if defined(__clang__)
```

Comment:
How about `__has_cpp_attribute(likely)` and C++20 [[likely]] support?

Reply:
It is C

Reply:
Argh...